### PR TITLE
Fix AudioCreateTranscriptionResponse JSON serialization

### PR DESCRIPTION
**Pull request by Swimburger**
The default value for `response_format` is `json` when omitted from the API call, so the response should be serialized from JSON when `audioCreateTranscriptionRequest.ResponseFormat` is `null`.

Fixes issue #221

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -360,4 +360,7 @@ MigrationBackup/
 .ionide/
 
 # Fody - auto-generated XML schema
-FodyWeavers.xsd
\ No newline at end of file
+FodyWeavers.xsd
+
```
Comment (kayhantolga) [inline .gitignore:364]: Was this change intentional? If so, could you provide more information about it? I couldn't find any details online regarding the change.

Comment (Swimburger) [reply .gitignore:364]: I don't know why "FodyWeavers.xsd" was modified, I only added the JetBrains IDE section.
.idea is where JetBrains IDEs like Rider store IDE specific settings and stuff.
